fragdenstaat_de/settings/base.py

```diff
@@ -49,7 +49,6 @@ def INSTALLED_APPS(self):
                 "djangocms_transfer",
                 # easy thumbnails comes from froide
                 "filer",
-                "mptt",
```

Comment:
Why is this needed? Are there any implications?

Reply:
This used to be a dependency of django-filer, see [upgrading notes](https://django-filer.readthedocs.io/en/latest/upgrading.html). Should have probably documented all these questions beforehand, sorry!